File: api/app/Models/Category.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Category extends Model
{
    protected $fillable = [
        'name',
        'background_color',
        'image_id'
    ];

    protected $hidden = [
        'created_at',
        'updated_at',
        'image_id'
    ];

    public function image()
    {
        return $this->belongsTo(Image::class);
    }
}

File: mobile_apps/RequestsLibrary/Models/Category.cs
using Newtonsoft.Json;

namespace RequestsLibrary.Models;
public class Category
{
    [JsonProperty("id")] public int Id { get; set; }

    [JsonProperty("name")] public string? Name { get; set; }

    [JsonProperty("background_color")] public string? BackgroundColor { get; set; }

    [JsonProperty("image")] public Image? Image { get; set; }
}
